website/docs/r/data_protection_backup_policy_postgresql.html.markdown

```diff
@@ -0,0 +1,135 @@
+}
+
+resource "azurerm_data_protection_backup_policy_postgresql" "example" {
+  name                = "example-backup-policy"
+  resource_group_name = azurerm_resource_group.rg.name
+  vault_name          = azurerm_data_protection_backup_vault.example.name
+
+  backup_repeating_time_intervals = ["R/2021-05-23T02:30:00+00:00/P1W"]
+
+  default_retention_duration = "P4M"
+
+  retention_rule {
+    name             = "weekly"
+    duration         = "P6M"
+    tagging_priority = 20
+    tagging_criteria {
+      absolute_criteria = "FirstOfWeek"
+    }
+  }
+
+  retention_rule {
+    name             = "thursday"
+    duration         = "P1W"
+    tagging_priority = 25
+    tagging_criteria {
+      days_of_week           = ["Thursday"]
+      scheduled_backup_times = ["2021-05-23T02:30:00Z"]
+    }
+  }
+
+  retention_rule {
+    name             = "monthly"
+    duration         = "P1D"
+    tagging_priority = 15
+    tagging_criteria {
+      weeks_of_month         = ["First", "Last"]
+      days_of_week           = ["Tuesday"]
+      scheduled_backup_times = ["2021-05-23T02:30:00Z"]
+    }
+  }
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name which should be used for this Backup Policy PostgreSQL. Changing this forces a new Backup Policy PostgreSQL to be created.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the Backup Policy PostgreSQL should exist. Changing this forces a new Backup Policy PostgreSQL to be created.
+
+* `vault_name` - (Required) The name of the Backup Vault where the Backup Policy PostgreSQL should exist. Changing this forces a new Backup Policy PostgreSQL to be created.
+
+* `backup_repeating_time_intervals` - (Required) Specifies a list of repeating time interval. It supports weekly back. It should follow `ISO 8601` repeating time interval. Changing this forces a new Backup Policy PostgreSQL to be created.
+  
+* `default_retention_duration` - (Required) The duration of default retention rule. It should follow `ISO 8601` duration format. Changing this forces a new Backup Policy PostgreSQL to be created.
+
+---
+
+* `retention_rule` - (Optional) One or more `retention_rule` blocks as defined below. Changing this forces a new Backup Policy PostgreSQL to be created.
+
+---
+
+A `retention_rule` block supports the following:
+
+* `name` - (Required) The name which should be used for this retention rule. Changing this forces a new Backup Policy PostgreSQL to be created.
+
+* `duration` - (Required) Duration after which the backup is deleted. It should follow `ISO 8601` duration format. Changing this forces a new Backup Policy PostgreSQL to be created.
+
+* `tagging_criteria` - (Required) A `tagging_criteria` block as defined below. Changing this forces a new Backup Policy PostgreSQL to be created.
+
+* `tagging_priority` - (Required) Retention Tag priority. Changing this forces a new Backup Policy Postgre Sql to be created.
```

Comment:
do we really need tagging here? it makes little sense and is confusing how about:
```suggestion
* `criteria` - (Required) A `tagging_criteria` block as defined below. Changing this forces a new Backup Policy PostgreSQL to be created.

* `priority` - (Required) Retention Tag priority. Changing this forces a new Backup Policy Postgre Sql to be created.
```

so we get retention.criteria

Reply:
Great, I'll modify it.